api/schema/resolvers/IssueResolver.js

```diff
@@ -13,8 +16,22 @@ module.exports = {
             return models.Issue.findByPk(id)
 
         },
-        getIssuesByProjectId: async (root, { project_id }, { models }) => {
-            return models.Issue.findAll({ where: { project_id } })
+        getIssuesByProjectId: async (root, { project_id, limit, offset, last_30_days_only }, { models }) => {
+            const issuesConditions = {
+                project_id
+            }
+            if (last_30_days_only) {
+                issuesConditions.created_at = {
+                    [Op.gte]: `${moment().subtract(30, 'days').toDate()}`
+                }
+            }
+            return models.Issue.findAll({
+                where: issuesConditions,
+                limit: limit,
+                offset: offset,
```

Comment:
Shouldn't we have defaults here if these are not provided? They would be `undefined` no?

Reply:
@Ovifer13 can you take a look at this PR and finish it if you have time?